src/api/app/controllers/status/checks_controller.rb

```diff
@@ -59,8 +59,13 @@ def require_or_initialize_checkable
   end
 
   def require_checkable
-    @checkable = Status::RepositoryPublish.find_by(build_id: params[:repository_publish_build_id]) if params[:repository_publish_build_id]
-    render_error(status: 404, errorcode: 'not_found', message: "Unable to find status_repository_publish with id '#{params[:repository_publish_build_id]}'") unless @checkable
+    if params[:repository_publish_build_id]
```

Comment:
Yes, but I think it is still not needed. `find_by` returns `nil` if you give it `nil` as a parameter. So this would do exactly the same if the `if` is not there. Conceptually also keeps the same.